week2/homework/src/index.js

```diff
@@ -1,3 +1,98 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+let fs = require('fs');
+let command = process.argv[2];
+let index = process.argv[3];
+let item = process.argv[4];
+
+switch (command) {
+    case 'list':
+        listValue()
+        break;
+
+    case 'add':
+        add(index);
+        break;
+
+    case 'remove':
+        Mydelete(index);
+        break;
+
+    case 'update':
+        updateValue(index, item)
+        break;
+
+    case 'reset':
+        Mydelete(0, 1);
+        break;
+
+    case 'help':
+    default:
+        printHelp();
+}
+
+function listValue() {
+    fs.readFile('./to-do.txt', 'utf8', function (error, data) {
+        if (error) {
+            console.log(error);
+        } else {
+            console.log(data);
+        }
+    })
+}
+
+function add(main) {
+    fs.appendFile('./to-do.txt', main + '\n', function (error) {
+        if (error)
+            console.log(error);
+    });
+}
+
+function Mydelete(index, reset) {
+    fs.readFile('to-do.txt', 'utf8', function (error, data) {
+        if (error) {
+            console.log(error);
+        }
+        else {
+            let corrected = data.split('\n');
+            corrected.splice(index - 1, 1);
+            corrected = corrected.join('\n');
+            if (reset)
+                corrected = "";
```

Comment:
It is good that you are making one function for similar functionality, you should however make this functionality more clear bij naming the function `removeItemOrClearList()` or something and don't pass it the number `1` as an argument, but just `true`, that will make it more clear what is going an. 
This is however a good mindset for a developer. You only have to improve it a little bit so that others also understand it when they read it.